source/frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -237,79 +238,88 @@ const Map: React.FC<React.PropsWithChildren<MapProps>> = ({
   const [layersOpen, setLayersOpen] = React.useState(false);
 
   return (
-    <Styled.MapGrid ref={resizeRef} className={classNames('px-0', 'map', { sidebar: showSideBar })}>
-      <LoadingBackdrop show={propertiesLoading || mapLoading} parentScreen />
-      {!showSideBar ? (
-        <StyledFilterContainer fluid className="px-0">
-          <PropertyFilter
-            useGeocoder={true}
-            defaultFilter={{
-              ...defaultFilterValues,
-            }}
-            onChange={handleMapFilterChange}
-            setTriggerFilterChanged={setTriggerFilterChanged}
-          />
-        </StyledFilterContainer>
-      ) : null}
-      <Styled.MapContainer>
-        {baseLayers?.length > 0 && (
-          <BasemapToggle baseLayers={baseLayers} onToggle={handleBasemapToggle} />
-        )}
-
-        <ReactLeafletMap
-          center={[lat, lng]}
-          zoom={lastZoom}
-          maxZoom={MAP_MAX_ZOOM}
-          closePopupOnClick={true}
-          ref={handleMapCreated}
-          whenReady={handleMapReady}
+    <VisibilitySensor partialVisibility={true}>
```

Comment:
when the sidebar is hidden, stop displaying map elements. Fixes issues with absolutely positioned items and causing the fullscreen sidebar to scroll.